### PR TITLE
Fixes #3841: 'time_ago' & quantity plurals should be combined

### LINKED ISSUE #3841: 'time_ago' & quantity plurals should be combined
After #3795, TextViewBindingAdapters combines two strings (time_ago & plurals) into a single one. This ought to just be different plural options so that translators have better context when translating the full conceptual string.

### LINKED ISSUE #3842: Investigate second/millisecond conversion check for greeting timestamp
After #3795, TextViewBindingAdapters performs a check on whether the last visited timestamp is in milliseconds or seconds. We ought to just use milliseconds & never need this conversion, but investigation is needed to better understand where this check came from & if it could ever be triggered.

### PR DESCRIPTION
**Pull request by mbobiosio**
## Explanation

Fixes #3841
Fixes #3842
This PR includes the required update to combine strings together

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

File: app/src/main/java/org/oppia/android/app/databinding/TextViewBindingAdapters.java [hunk 1]
```diff
@@ -107,24 +106,11 @@ private static String getPluralString(
       @PluralsRes int pluralsResId,
       int count
   ) {
-    // TODO(#3841): Combine these strings together.
-    return resourceHandler.getStringInLocaleWithWrapping(
-        R.string.time_ago,
-        resourceHandler.getQuantityStringInLocaleWithWrapping(
-            pluralsResId, count, String.valueOf(count)
-        )
+    return resourceHandler.getQuantityStringInLocaleWithWrapping(
```
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/databinding/TextViewBindingAdapters.java:109]: Isn't this change going to remove the time_ago text altogether? I'm not sure that's what we want without the code that combines plurals. Is there a reason that was removed in this new PR?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @mbobiosio. I'm a bit confused since this PR seems incomplete as changes that were previously here are no longer here.

Also, as mentioned in #4560 please include relevant screenshots in the PR description to demonstrate the visual changes introduced as part of this PR.

File: app/src/main/java/org/oppia/android/app/databinding/TextViewBindingAdapters.java (hunk 1 above)
Comment (mbobiosio) [reply app/src/main/java/org/oppia/android/app/databinding/TextViewBindingAdapters.java:109]: I figured yesterday there's this problem from CI tests. I'll work to resolve this.

**Comment by BenHenning** [comment]
@mbobiosio are you working on this still?

**Pull request closed without merging**